expr/function/math.go

```diff
@@ -0,0 +1,219 @@
+package function
+
+import (
+	"math"
+
+	"github.com/brimsec/zq/anymath"
+	"github.com/brimsec/zq/expr/coerce"
+	"github.com/brimsec/zq/expr/result"
+	"github.com/brimsec/zq/zng"
+)
+
+type abs struct {
+	result.Buffer
+}
+
+func (a *abs) Call(args []zng.Value) (zng.Value, error) {
+	v := args[0]
+	id := v.Type.ID()
+	if zng.IsFloat(id) {
+		f, _ := zng.DecodeFloat64(v.Bytes)
+		f = math.Abs(f)
+		return zng.Value{zng.TypeFloat64, a.Float64(f)}, nil
+	}
+	if !zng.IsInteger(id) {
+		return badarg("Math.abs")
```

Comment:
Is this the time to update errors to use the new names?

Reply:
Okay, I changed the ones that have new ones and left the ones that will be deleted or merged to a later PR.